input/config/rocky/8.8/telemetry.json

```diff
@@ -1,211 +1,248 @@
+      },
+      {
+        "package": "nfs-subdir-external-provisioner-4.0.18",
+        "type": "tarball",
+        "url": "https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/releases/download/nfs-subdir-external-provisioner-4.0.18/nfs-subdir-external-provisioner-4.0.18.tgz"
+      },
+      {
+        "package": "docker.io/library/nginx",
+        "tag": "1.25.2-alpine",
+        "type": "image"
+      },
+      {
+        "package": "registry.k8s.io/coredns/coredns",
+        "tag": "v1.11.1",
+        "type": "image"
+      },
+      {
+        "package": "registry.k8s.io/cpa/cluster-proportional-autoscaler",
+        "tag": "v1.8.8",
+        "type": "image"
+      },
+      {
+        "package": "registry.k8s.io/dns/k8s-dns-node-cache",
+        "tag": "1.22.28",
+        "type": "image"
+      },
+      {
+        "package": "registry.k8s.io/kube-apiserver",
+        "tag": "v1.29.5",
+        "type": "image"
+      },
+      {
+        "package": "registry.k8s.io/kube-controller-manager",
+        "tag": "v1.29.5",
+        "type": "image"
+      },
+      {
+        "package": "registry.k8s.io/kube-proxy",
+        "tag": "v1.29.5",
+        "type": "image"
+      },
+      {
+        "package": "registry.k8s.io/kube-scheduler",
+        "tag": "v1.29.5",
+        "type": "image"
+      },
+      {
+        "package": "registry.k8s.io/pause",
+        "tag": "3.9",
+        "type": "image"
+      },
+      {
+        "package": "quay.io/coreos/etcd",
+        "tag": "v3.5.12",
+        "type": "image"
+      },
+      {
+        "package": "quay.io/calico/cni",
+        "tag": "v3.27.3",
+        "type": "image"
+      },
+      {
+        "package": "quay.io/calico/kube-controllers",
+        "tag": "v3.27.3",
+        "type": "image"
+      },
+      {
+        "package": "quay.io/calico/pod2daemon-flexvol",
+        "tag": "v3.27.3",
+        "type": "image"
+      },
+      {
+        "package": "quay.io/calico/node",
+        "tag": "v3.27.3",
+        "type": "image"
+      },
+      {
+        "package": "docker.io/flannel/flannel-cni-plugin",
+        "tag": "v1.1.2",
+        "type": "image"
+      },
+      {
+        "package": "docker.io/flannel/flannel",
+        "tag": "v0.22.0",
+        "type": "image"
+      },
+      {
+        "package": "quay.io/metallb/speaker",
+        "tag": "v0.13.9",
+        "type": "image"
+      },
+      {
+        "package": "quay.io/metallb/controller",
+        "tag": "v0.13.9",
+        "type": "image"
+      },
+      {
+        "package": "docker.io/kubernetesui/dashboard",
+        "tag": "v2.7.0",
+        "type": "image"
+      },
+      {
+        "package": "docker.io/kubernetesui/metrics-scraper",
+        "tag": "v1.0.8",
+        "type": "image"
+      },
+      {
+        "package": "docker.io/grafana/grafana-enterprise",
+        "tag": "8.3.2",
+        "type": "image"
+      },
+      {
+          "package": "kube-prometheus-stack-62.3.0",
```

Comment:
Check extra indentation here.

Reply:
Addressed changes in internal PR. It will be available in next set of PRs